src/lib/sql/sql_to_ast_translator.cpp

```diff
@@ -188,6 +188,7 @@ std::shared_ptr<AbstractASTNode> SQLToASTTranslator::_translate_insert(const hsq
 
 std::shared_ptr<AbstractASTNode> SQLToASTTranslator::_translate_delete(const hsql::DeleteStatement& del) {
   std::shared_ptr<AbstractASTNode> current_result_node = std::make_shared<StoredTableNode>(del.tableName);
+  current_result_node = _validate_if_active(current_result_node);
```

Comment:
Can somebody explain the reasoning behind this method name? What does `active` mean in this context? If validation is activated?

Reply:
I had my problems with it, too. It adds a validate node if the SQLToASTTranslator is configured to add validate. It's immediately clear when you look into the method, but the name is not the most speaking one.

Reply:
Hm, `validate_if_validation_configured` or `desired` makes the method name only longer and does not really help. I guess we will leave it like it is for now?